toolchain/sem_ir/file.h

```diff
@@ -268,8 +268,8 @@ class File : public Printable<File> {
   auto reals() const -> const ValueStore<RealId>& {
     return value_stores_->reals();
   }
-  auto strings() -> ValueStore<StringId>& { return value_stores_->strings(); }
-  auto strings() const -> const ValueStore<StringId>& {
+  auto strings() -> StringStore& { return value_stores_->strings(); }
+  auto strings() const -> const StringStore& {
     return value_stores_->strings();
   }
```

Comment:
While I think it is reasonable to share the storage and the ID space here, I wonder if this is the interface we want -- in particular, do we want to expose the fact that we store IDs and string literals in the same storage to API clients?

We could instead provide two wrappers around the internal value store, one for string literals and one for identifiers, so that client code can use `context.identifiers().Get(ident_id)` or `context.string_literals().Add("blah")` without needing to know that they share storage.

What do you think? I don't think it's essential to hide this implementation detail, but it would seem a bit more consistent to me to do so, so that each kind of ID has its own function to access the value store.

Reply:
Done